tests/common2/README.md

```diff
@@ -0,0 +1,57 @@
+# Common2: A Cleaner and Redesigned Test Utilities Directory
+
+The `common2` directory is a redesigned and cleaner version of the existing `common` directory. It aims to improve the structure, maintainability, and usability of shared utility functions, fixtures, and helpers used across the SONiC test suite.
+
+## Purpose
+
+The `common2` directory is being developed to address the following goals:
+
+1. **Code Redesign**: Refactor and redesign existing utilities, fixtures, and helpers to improve readability, modularity, and maintainability.
+2. **Directory Restructuring**: Organize the codebase into a more logical and intuitive directory structure.
+3. **Code Quality Enforcement**: Ensure high-quality code by enforcing pre-commit checks, including:
+   - **Linter checks** using `pylint`
+   - **Type checking** will be enforced with `mypy`
+   - **Code formatting** done using `black`
+   - **Unit Testing**: Every API library function added to `common2` will be accompanied by unit tests to ensure correctness, maintainability, and ease of debugging.
```

Comment:
Is any of the check able to cover "docstring"? We need to enforce well formatted docstring. Then we can use tools to generate html docs for the code under `common2`.

Reply:
I have added checks for docstring in pylintrc in https://github.com/sonic-net/sonic-mgmt/pull/18220